.github/workflows/python-package.yml

```diff
@@ -202,8 +205,6 @@ jobs:
         with:
           name: delocate-sdist
           path: dist/
-      - name: Upload to PyPI
-        env:
-          TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-        run: twine upload dist/* --non-interactive --skip-existing
+      - uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          skip-existing: true
```

Comment:
I usually consider this feature harmful — are there multiple upload sources?

Reply:
No, that was just habit, and the previous script was already using `--skip-existing`, and my other projects I was referencing were also using it.

It'd be reasonable to remove it.